packages/react-examples/src/react-charting/GaugeChart/GaugeChartPage.tsx

```diff
@@ -47,36 +47,68 @@ export class GaugeChartPage extends React.Component<IComponentDemoPageProps, {}>
               left to right.
             </p>
             <h3>Implementation details</h3>
+            <div>Here are the props available for customizing the gauge chart:</div>
             <ul>
               <li>
-                The diameter of the gauge depends upon the <code>width</code> and <code>height</code> props passed to
-                the chart. If the props are omitted, a default diameter of 140px will be used.
+                <code>width</code> and <code>height</code>: These props determine the diameter of the gauge. If not
+                provided, a default diameter of 140px is used.
               </li>
               <li>
-                To render a title above the gauge, set the <code>chartTitle</code> prop.
+                <code>chartTitle</code>: Use this prop to render a title above the gauge.
               </li>
               <li>
-                The needle position depends upon the required <code>chartValue</code> prop.
+                <code>chartValue</code>: This required prop controls the rotation of the needle. If the chart value is
+                less than the minimum, the needle points to the min value. Similarly, if it exceeds the maximum, the
+                needle points to the max value.
               </li>
               <li>
-                Use the required <code>segments</code> prop to divide the gauge into colored sections. These sections
-                can have fixed sizes, or the users can choose to create a sweeping effect by varying the segment size
-                with the chartValue.
+                <code>segments</code>: Use this required prop to divide the gauge into colored sections. The segments
+                can have fixed sizes or vary with the chart value to create a sweeping effect. Negative segment sizes
+                are treated as 0.
               </li>
               <li>
-                Set the <code>minValue</code> prop if the minimum value of the gauge is different than 0. A placeholder
-                segment will be rendered if the <code>maxValue</code> prop is greater than the total size of the
-                segments.
+                <code>minValue</code>: Use this prop if the minimum value of the gauge is different from 0.
               </li>
               <li>
-                To render an additional text below the chartValue, set the <code>sublabel</code> prop.
+                <code>maxValue</code>: Use this prop to render a placeholder segment when the difference between the max
+                and min values is larger than the total size of the segments. If the difference is smaller, the max
```

Comment:
rephrase to - when the desired range for the gauge is more than the sum of all segments.
If the maxValue is less than the sum of all segments, this property is ignored.

Reply:
done